encoding/da.go

```diff
@@ -509,6 +509,15 @@ func checkCompressedDataCompatibilityV7(data []byte) error {
 	return nil
 }
 
+// Fast testing if the compressed data (v8) is compatible with our circuit
+// (require specified frame header and each block is compressed)
+func checkCompressedDataCompatibilityV8(data []byte) error {
```

Comment:
Currently changing this function, only keeping a very simple check (other checks are failing in unit tests). cc @noel2004 to check if it's safe to do this.